plugins/formal-verification/testData/diagnostics/bad_contracts/cond_effects.fir.diag.txt

```diff
@@ -11,9 +21,9 @@ method global$fun_mayReturnNonNull$fun_take$NT_Any$return$NT_Any(local$x: dom$Nu
   label label$ret$0
 }
 
-/cond_effects.kt:(178,210): warning: Viper verification error: Postcondition of global$fun_mayReturnNonNull$fun_take$NT_Any$return$NT_Any might not hold. Assertion ret$0 == (dom$Nullable$null(): dom$Nullable[dom$Any]) ==> dom$Type$isSubtype((dom$TypeOf$typeOf(local$x): dom$Type), dom$Type$Int()) might not hold.
+/cond_effects.kt:(328,360): warning: Cannot verify that if a null value is returned then 'x is kotlin.Int'.
```

Comment:
It seems a bit distracting to use the fully qualified type here, WDYT?

Reply:
I agree! I should check if we have a renderer for just the type name without all the rest